Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM alpine
```

Comment:
I suggest we do the same thing we did for mattermod ([here](https://github.com/mattermost/mattermost-mattermod/blob/master/Dockerfile)). With this approach, the `make docker` won't work. It needs `bifrost` existed and built beforehand.

Reply:
Yeah is there any benefit to build the app inside a container? And `make docker` already includes the `make build` so it should work, or am I missing something?

Reply:
With multi-staged Dockerfile we would get a standalone build without needing to know any other part of the build process. But on the otherhand building the binary outside of the docker itself potentially will speed up the build time and using the go caching mechanism. I'm fine either way to be honest.

Reply:
Oh I see, at the moment I believe the app is simple enough to either build inside/outside of docker. We can adopt that approach when it is necessary. Thanks for pointing out and for the explanation 👍